lib/global-styles.php

```diff
@@ -135,9 +136,9 @@ function gutenberg_experimental_global_styles_get_stylesheet( $tree ) {
 		}
 	}
 
-	$stylesheet = $tree->get_stylesheet();
+	$stylesheet = $tree->get_stylesheet( $type );
```

Comment:
So, now, the resulting stylesheet can be actually 3 different stylesheets, however, we only have 1 cache. I presume that can be a source of conflicts we need to fix?

Reply:
Nice catch we should only cache type all which is the one used on the front end.